### ISSUE TITLE
Translations for offline docs

### ISSUE THREAD

**Issue by sezanzeb** [sezanzeb/key-mapper#191]
After the macro system got more features it also got some [improved documentation](https://github.com/sezanzeb/key-mapper/blob/clean-history-main/readme/macros.md). Instead of replicating that in the GUI I'm linking that in the help window right now

Should there be offline docs that can be translated instead, or is linking the English markdown file fine?

It might be possible to render the markdown files into the application with something like https://github.com/ubunatic/md2pango, then I guess it should be possible to translate them for the GUI

![image](https://user-images.githubusercontent.com/28510156/136954074-9081b2aa-4522-42fe-8387-5bbab41ce0e4.png)

Do you have an opinion on how to handle documentation in the GUI in the future @nlepage, @albanobattistella, @jose1711?

Thanks!

**Comment by albanobattistella** [comment]
> Adding translation of docs sounds like a good idea. And frankly I am OK with whatever format you decide on.

I agree with you.  @nlepage I prefer also a well maintained English documentation.
